lms/djangoapps/courseware/views/views.py

```diff
@@ -198,9 +198,7 @@ def jump_to_id(request, course_id, module_id):
 def jump_to(_request, course_id, location):
     """
     Show the page that contains a specific location.
-
```

Comment:
Unsure why these lines are being removed?  They weren't removed in the original [PR#16671](https://github.com/edx/edx-platform/pull/16671/files#diff-2749747f3edc5443c734c6a6546e91d1), and aren't removed in [upstream master](https://github.com/edx/edx-platform/blob/master/lms/djangoapps/courseware/views/views.py#L260) either.

Could you please un-delete them to keep the diff small?  Same comment for all of the empty lines removed in this file.